src/content/learning/using-durable-objects.md

```diff
@@ -0,0 +1,216 @@
+## Instantiating a Durable Object
+
+Creating a Durable Object requires retrieving a Fetcher object from the class in your Workers Script.  Once you have the Fetcher object, you can use it to send [Requests](/runtime-apis/request) to the Durable Object instance.
+
+```js
+
+addEventListener('fetch', event => {
+  event.respondWith(handleRequest(event.request))
+})
+ 
+async function handleRequest(request) {
+  // Retrieve the Fetcher for the Durable Object with id 1
+  let durableObject = await DURABLE_OBJECT_CLASS.get("1")
+  // Send a request to Durable Object '1', with the URI "get-count"
+  let resp = await durableObject.fetch("get-count")
+  let count = await resp.text()
+ 
+  // Regardless of the URI, requests are always routed to the Durable Object
+  resp = await durableObject.fetch()
+  let message = await resp.text()
+
+  // TODO: system generated id example
+  let id = generateId()
+  let sysDurableObject = await DURABLE_OBJECT_CLASS.get()
+  resp = await sysDurableObject.fetch('get-count')
+  let sysCount = await resp.text()
+ 
+  return new Response("Object '1' count: " + count + " and message " + message + ".  System Object '" + id + "' count: " + sysCount)
+}
+
+```
+
+Learn more at the [Workers Durable Objects API reference](/runtime-apis/durable-objects).
+
+## Limitations and considerations when working with Durable Objects
+
+### Global Uniqueness
+
+Uniqueness is currently enforced only when a Durable Object begins execution.  In the future, uniqueness will also be enforced whenever a Durable Object makes storage requests.  
+
+During a network partition between data centers or between machines within a data center, it is possible for more than one Durable Object with a given id to be active.  While this is unlikely, we plan to do additional work to reduce the chance of a network partition during the beta period.
+
+### Creation and Routing
+
+Durable Objects use a deterministic hash of the Object's id to determine the Cloudflare point-of-presence that owns the Object.  It is not currently possible for an Object to migrate ownership to a different point-of-presence.
+
+When using system generated ids, Durable Objects are created in a Cloudflare point-of-presence located near the Worker that made the creation request.
+
+When using client-defined ids, Durable Objects may be created in any of Cloudflare's points-of-presence. Durable Object with client-defined ids will see greater variance in request latency across different Objects, as each Object may be routed to a different point-of-presence.  
+
+When migrations of Objects are enabled, this variance in request latency will only affect Object creation.
+
+### Transactional Storage API
+
+The transactional storage API is scoped to a single Durable Object.  It is not currently possible to access data stored in a Durable Object from a different Durable Object directly through the API.
+
+There is no support for querying, bulk loading, or exporting data stored in Durable Objects.
```

Comment:
"querying" is such a vague word, and it's not that it isn't possible to load or export data so much as that you have to write your own code to do it for now. If you don't mind all the edits, I'd make this sentence:

"There is no support for advanced queries or bulk loading or exporting of data. Loading or exporting data can be done by writing custom Workers and Durable Objects to do so yourself.